doc/source/development/contributing.rst

```diff
@@ -949,6 +949,15 @@ the expected correct result::
 
         assert_frame_equal(pivoted, expected)
 
+.. note::
+ 
+   Add the GitHub issue number as a comment to the new tests.
+   For Example - For a test solving issue number 20172, write  
+     | # GH 20172 
```

Comment:
Should move this to the function body. Check the test as an example in #28662 if needed